src/main/scheduler/scheduler.h

```diff
@@ -189,11 +189,11 @@ typedef struct {
     void (*taskFunc)(timeUs_t currentTimeUs);
     timeDelta_t desiredPeriodUs;        // target period of execution
     const int8_t staticPriority;        // dynamicPriority grows in steps of this size
-} task_attr_t;
+} task_attributes_t;
 
 typedef struct {
     // Task static data
-    task_attr_t *attr;
+    task_attributes_t *attributes;
```

Comment:
```suggestion
    task_attributes_t attributes;
```

Reply:
@ledvinap I've tried to implement but got a lot of errors doing the suggestions.

I've ended here:
[diff.txt](https://github.com/betaflight/betaflight/files/7848213/diff.txt)

Reply:
Thanks to @daleckystepan for pointing out the following: 
```
typedef struct {
    // Configuration
    const char * taskName;
    const char * subTaskName;
    bool (*checkFunc)(timeUs_t currentTimeUs, timeDelta_t currentDeltaTimeUs);
    void (*taskFunc)(timeUs_t currentTimeUs);
    timeDelta_t desiredPeriodUs;        // target period of execution
    int8_t staticPriority;        // dynamicPriority grows in steps of this size
} task_attributes_t;
```
staticPriority was defined as const and assignment violated that.

Reply:
@ledvinap Implemented your suggestion in second commit.